Makefile

```diff
@@ -94,6 +94,8 @@ core:
 	$(MAKE) minimal-build
 	# make executables easily accessible for manual testing:
 	ln -s semgrep-core bin/osemgrep
+	chmod +w _build/install/default/bin/semgrep-core
+	strip _build/install/default/bin/semgrep-core
```

Comment:
I'd like to add a comment that links to this PR. Something like this would be useful to future maintainers:
```
# Remove all symbols with GNU strip. It saves 12-21% on the executable size
# and it doesn't seem to reduce the functionality or debuggability of OCaml
# executables. See https://github.com/semgrep/semgrep/pull/9471
```

Another thing: I would run `strip bin/semgrep-core` since it's a simpler path that's human-friendly. `strip` follows symlinks and edits the executable in place.

(I might do this myself after merging, we'll see)

Reply:
Thanks for the feedback, I'll add the comment.
Regarding the simpler path, that was my first (or one of the first) but, while it worked locally, it kept erroring in the GitHub action saying path wasn't found, even though the line before specially used it
But I'll retry.

Reply:
Not sure what's going on with the GitHub checks. I'll take it from here -> https://github.com/semgrep/semgrep/pull/9765
Thanks for your contribution!

Reply:
Do you mind if I continue to it?
There's a certain satisfaction of getting a "PR merged" notification plus commit authorship 😄

Reply:
Merged.

Reply:
Lovely